worker-markup-container-fs/pom.xml

```diff
@@ -233,7 +233,7 @@
                     <watchInterval>500</watchInterval>
                     <logDate>default</logDate>
                     <verbose>true</verbose>
-                    <autoPull>always</autoPull>
+                    <autoPull>on</autoPull>
```

Comment:
Yeah this is so that it uses the jep-talon image that was built as part of the current build run rather than pulling an older version of the image from a remote location.